groovy/inductor_locally_benchmark.groovy

```diff
@@ -46,7 +46,7 @@ if (env.NODE_LABEL == "0") {
     if (env.precision == "float32") {
         env.NODE_LABEL = "inductor-icx-local-tas"
     } else if (env.precision == 'amp') {
-        env.NODE_LABEL = "inductor-gnr-local-tas"
+        env.NODE_LABEL = "inductor-gnr-local-tas-sh"

```

Comment:
can we launch the test use a specific node? e.g. can I run Fp32 test on a GNR node?

Reply:
I was wondering it might be easy to make mistake that if they forget to set both of them. Like run AMP on icx or run FP32 on GNR. So we only keep 1 parameter to control. 
Currently, they did not have requirement for FP32 on GNR node case. If they need, it's easy to enable.